tests/libcxx/expected_results.txt

```diff
@@ -38,6 +38,10 @@ std/containers/sequences/vector.bool/vector.bool.fmt/parse.pass.cpp FAIL
 std/thread/thread.threads/thread.thread.class/thread.thread.id/parse.pass.cpp FAIL
 std/utilities/format/format.tuple/parse.pass.cpp FAIL
 
+# We should provide 'test_format_context_create' function in 'libcxx/test/support/test_format_context.h' file to enable those tests.
+std/utilities/format/format.range/format.range.formatter/set_brackets.pass.cpp FAIL
+std/utilities/format/format.range/format.range.formatter/set_separator.pass.cpp FAIL
+

```

Comment:
There's a pre-existing section for `test_format_context_create` function in `libcxx/test/support/test_format_context.h`. Presumably we should merge both sections.
https://github.com/microsoft/STL/blob/8dc4faadafb52e3e0a627e046b41258032d9bc6a/tests/libcxx/expected_results.txt#L1106-L1127